exonum/src/sandbox/consensus/mod.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// spell-checker:ignore unsynchronized
+
+mod basic;
+mod block_request;
+mod config;
+mod invalid_message;
+mod recovery;
+mod round_details;
+mod timeouts;
+mod transactions;
+mod unsynchronized_message;
+
+// - lock to propose when get +2/3 prevote
```

Comment:
Do we need those comments?

Reply:
Ideally, only documentation for each test function + module level doc should be enough. 
But about this one, I am not sure since it is sort of high-level overview of the tests.